caffe_app/tests.py

```diff
@@ -1 +1,16 @@
-# Create your tests here.
+from django.test import Client
```

Comment:
Can we have imports grouped in alphabetical order? For reference, see this https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement?answertab=active#tab-top